itests/pom.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>admin</artifactId>
+        <groupId>org.codice.ddf.admin.beta</groupId>
+        <version>0.1.3-SNAPSHOT</version>
+    </parent>
+    <name>DDF :: Admin (Beta) :: Itests</name>
+    <artifactId>itests</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <pax.tinybundles.version>2.1.1</pax.tinybundles.version>
+        <pax.url.version>2.4.5</pax.url.version>
+        <pax.exam.version>4.11.0</pax.exam.version>
+    </properties>
+
+    <dependencies>
+        <!--start pax-exam dependencies-->
+        <dependency>
+            <groupId>org.apache.karaf.bundle</groupId>
+            <artifactId>org.apache.karaf.bundle.core</artifactId>
+            <version>${karaf.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.exam</groupId>
+            <artifactId>pax-exam-spi</artifactId>
+            <version>${pax.exam.version}</version>
+            <scope>test</scope>
```

Comment:
Nothing else in here is marked as test scope. Should it be? Or should this _not_ be scoped as test?